### DISCUSSION TITLE
What is the purpose of the connection to posthog.com? Do I need to unblock that?

### DISCUSSION THREAD

**Discussion by TiagoTiago** [Josh-XT/AGiXT#279] [Q&A] [answered]
Are there any preparations required to be able to run this fully offline?

**Comment by Josh-XT** [comment]
I don't know, I assume it is happening from whichever provider you're using.

**Reply by TiagoTiago** [reply]
I'm using Oobabooga, and it doesn't try that connection on it's own.

**Reply by Josh-XT** [reply]
I'm not sure which module is using it, but we're not doing anything with posthog in the Agent-LLM code.

**Reply by TiagoTiago** [reply, marked as answer]
Alright, I'll leave it blocked then.
